geoips/plugins/yaml/sectors/dynamic/tc_huge/tc_1km_3200x3200.yaml

```diff
@@ -11,7 +11,7 @@ spec:
       name: tc
       arguments: {}
     spec:
-      name: clat_clon_resolution_shape
+      name: center_coordinates
```

Comment:
@jsolbrig Note I updated these (because a few tests are actually using the new dynamic sector templates)

Reply:
Looks good, thanks!